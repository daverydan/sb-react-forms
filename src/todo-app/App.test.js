import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('app not broken', () => {
  render(<App />);
});

test("it matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});