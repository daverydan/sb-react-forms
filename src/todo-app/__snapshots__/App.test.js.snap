// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Todo App
    </h1>
    <form
      class="NewTodoForm"
    >
      <label
        for="todo"
      >
        Add Task
      </label>
      <input
        id="todo"
        name="todo"
        type="text"
        value=""
      />
      <button>
        Add
      </button>
    </form>
    <hr />
    <div
      class="TodoList"
    >
      <div
        class="Todo"
      >
        clean room 
        <button>
          x
        </button>
      </div>
      <div
        class="Todo"
      >
        eat lunch 
        <button>
          x
        </button>
      </div>
      <div
        class="Todo"
      >
        take nap 
        <button>
          x
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
